### ISSUE TITLE
Any way to allow connection after locast shutdown so we can remove the DVR from Plex?

### ISSUE THREAD

**Issue by wvdude** [wouterdebie/locast2tuner#107]
After Locast shut down, my locast2tuner docker will not run. I beleive it may be because it cannot login to locast. In plex when I click on settings for Live TV & DVR, I get "There was an unexpected error". That's it. I cannot do anything right now with that feature of plex. Is there a known fix for this? I am trying to use xTeVe. Thank you.

**Comment by eiddor** [comment]
Even with the container shutdown, you should be able to delete it.  When I go to Settings | Live TV & DVR (under Manage), I see this and can delete the tuner.

![image](https://user-images.githubusercontent.com/7586731/132106447-49d0b21e-0478-4bd1-ae3e-2ff08d79a59d.png)

**Comment by wvdude** [comment]
Thank you - I suppose it is a different problem then. I still just get an
error with no information. I appreciate your help.

**Comment by eiddor** [comment]
Yeah, Plex seems to be unhappy - I'd try restarting the server and then checking.

**Comment by rtm2017** [comment]
@wvdude, I just had a similar problem. Logging into my server remotely through the Plex website resulted in an error and unable to do anything on the live TV settings page, Once I logged into my Plex server locally using it's ip address the page worked correctly and i was able to delete the DVR.

**Comment by jamesmalachyryan** [comment]
I have the same problem.  Just happen when I started working with xTeVe.  It seems the combination of locast shutting down and using eTeVe create the issue.  Any solutions?
